src/components/Tag/__test__/index.test.tsx

```diff
@@ -0,0 +1,13 @@
+import { mount } from 'enzyme'
+import React from 'react'
+
+import { Tag } from '../Tag'
+
+describe('Tag', () => {
+  it('should render given children', () => {
+    const hello = 'hello'
+    const wrapper = mount(<Tag type="success">{hello}</Tag>)
+
+    expect(wrapper.text()).toBe(hello)
+  })
```

Comment:
Add snapshot test to reference `src/components/Checkbox/__test__/index.test.tsx`.